CONTRIBUTION.md

```diff
@@ -0,0 +1,101 @@
+# Contribute to DB4S
+
+Thanks for your interest in our project.  
+This document is intended to serve as lighthouse for contributors, not an executor.
+
+## Contents
+- [How to ask about DB4S](#how-to-ask-about-db4s)
+- [How to contribute?](#how-to-contribute)
+    - [Report bug](#report-bug)
+    - [Suggest enhancement](#suggest-enhancement)
+    - [Translation](#translation)
+- [Styleguide](#styleguide)
+    - [CPP source code](#cpp-source-code)
+    - [Git commit messeage](#git-commit-message)
+
+## How to ask about DB4S?
+You have questions about our project! If so, please check the following first.
+
+- Look up related topics on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki)
+- Search for existing similar issues
+
+If your question is still not resolved, you can create an issue and ask a question.  
+And also a template from GitHub issue will help you write good bug report.
+
+## How to contribute?
+### Report bug
+Please check the following before reporting a bug.
+
+- Look up ['Known Issues'](https://github.com/sqlitebrowser/sqlitebrowser/issues/1688)
+- Search for existing similar issues
+- Try in nightly build
+
+If a bug has not yet been repoted or has not been resolved, please follow the below to help members identify the bug.
+
+- Use a good title to identify the bug
+    - Ex) `Crash DB4S when try to reset the settings`
+- Detailed steps to reproduce the bug
+    - Ex) `File -> Preferences -> General -> Restore Defaults`
+- What you actually expected to do and why
+    - Ex) `DB4S settings are initalized normally.`
+- System environment, DB4S version, Loaded extensions
+    - Ex) `macOS Catalina Version 10.15.6(19G2021), DB4S Version 3.12.0, No Extensions`
+- Attach screenshots or screencasts related to the bug
+- Crash log (if the app is forced to close)
+    - If you have a long crash log, [Pastebin](https://pastebin.com/) can help
+- Project file or database file what used (**NOT** required)
+
+And also a template from GitHub issue will help you write good bug report.
+
+### Suggest enhancement
+Please check the following before suggest enhancement
+
+- Search for existing simillar issues
+
+And the please follow the below to help members identify the suggest.
```

Comment:
Replace with "And please follow the below to help members identify the suggestion."